src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Core/Services/Contracts/IAssignmentService.cs

```diff
@@ -58,13 +58,38 @@ public interface IAssignmentService
     /// Adds a package to the delegation
     /// </summary>
     /// <returns></returns>
-    Task<bool> AddPackageToAssignment(Guid userId, Guid assignmentId, Guid packageId, CancellationToken cancellationToken = default);
+    Task<bool> AddAssignmentPackage(Guid userId, Guid assignmentId, Guid packageId, CancellationToken cancellationToken = default);
 
     /// <summary>
     /// Adds a resource to the delegation
     /// </summary>
     /// <returns></returns>
-    Task<bool> AddResourceToAssignment(Guid userId, Guid assignmentId, Guid resourceId, CancellationToken cancellationToken = default);
+    Task<bool> AddAssignmentResource(Guid userId, Guid assignmentId, Guid resourceId, string policyPath, string policyVersion, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Adds a resource to the delegation
+    /// </summary>
+    /// <returns></returns>
+    Task<bool> AddAssignmentInstance(Guid userId, Guid assignmentId, Guid resourceId, string instanceId, string policyPath, string policyVersion, CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Adds a package to the delegation
+    /// </summary>
+    /// <returns></returns>
+    Task<bool> RemoveAssignmentPackage(Guid userId, Guid assignmentId, Guid packageId, CancellationToken cancellationToken = default);
+   
+    /// <summary>
+    /// Adds a resource to the delegation
+    /// </summary>
+    /// <returns></returns>
+    Task<bool> RemoveAssignmentResource(Guid userId, Guid assignmentId, Guid resourceId, CancellationToken cancellationToken = default);
```

Comment:
AssignmentResource og AssignmentInstance må også kunne oppdateres med ny policyPath og policyVersion for når det skjer endringer på enkelt tilganger (lagt til/fjernet enkelt actions f.eks)